src/vs/workbench/contrib/terminal/browser/terminalEditorService.ts

```diff
@@ -55,6 +58,46 @@ export class TerminalEditorService extends Disposable implements ITerminalEditor
 		}));
 	}
 
+	private _getActiveTerminalEditors(): IEditorInput[] {
+		return this._editorService.visibleEditors.filter(e => e instanceof TerminalEditorInput && e.terminalInstance?.instanceId);
+	}
```

Comment:
We already track this in `this._editorInputs`, this could be something like:

```ts
Array.from(this._editorInputs.values());
```

Reply:
we do not and cannot track the ones that have been added via copy